ml-conversational-analytic-tool/githubDataExtraction.py

```diff
@@ -161,17 +155,26 @@ def getPullFeatures(self, pull):
     parser = argparse.ArgumentParser(description='Create csv for all pulls in repo')
     parser.add_argument('organization', help='Organization the repo belongs to.')
     parser.add_argument('reponame', help='Name of repo')
-    parser.add_argument('-reactions', action='store_true', default=False, help='Flag to extract reactions')
-    parser.add_argument('--filename', help='Name of file')
-    # parser.add_argument('accesstoken', help='Github access token')
+    parser.add_argument('--reactions', action='store_true', default=False, help='Flag to extract reactions')
+    parser.add_argument('-n', "--name", help='Output file name. If not specified, the name is constructed like this: '
+            '<organization><reponame>.csv')
 
     args = parser.parse_args()
     ACCESS_TOKEN = os.environ["GITACCESS"]  # Access Github token from environment for security purposes
     extractor = GithubDataExtractor(ACCESS_TOKEN)  # Create object
-    extractor.openRepo(args.organization, args.reponame)  # Open repo
 
-    # Extract all pulls and export them to .csv
-    if args.filename:
-        extractor.getAllPulls(args.filename, args.reactions)
+    if args.reponame == 'all':
+        # Extract data for all repositories in organization
+        repos = extractor.g_ses.get_organization(args.organization).get_repos()
+        for repo in repos:
+            print("Starting: {}".format(repo.name))
+            extractor.openRepo(args.organization, repo.name)
+            df = extractor.getAllPulls(args.reactions)
+            file_name = utils.construct_filename(args.name, args.organization, args.reponame)
+            utils.export_to_cvs(df, file_name)
     else:
-        extractor.getAllPulls("", args.reactions)
+        # Extract data for an individual repository
+        extractor.openRepo(args.organization, args.reponame)  # Open repo
+        df = extractor.getAllPulls(args.reactions)
+        file_name = utils.construct_filename(args.name, args.organization, args.reponame)
```

Comment:
function with name `construct_filename` does not exists. The correct spelling is `construct_file_name`

Reply:
Done.